masterlist.yaml

```diff
@@ -21511,6 +21511,104 @@ plugins:
+        name: 'HS Riverwood - Faendal''s House'
+  - name: 'HSRiverwood - Hod and Gerdur''s House.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/74491/'
+        name: 'HS Riverwood - Hod and Gerdur''s House'
+  - name: 'HSRiverwood - Riverwood Trader.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/86090/'
+        name: 'HS Riverwood - Riverwood Trader'
+  - name: 'HSRiverwood - Sleeping Giant Inn.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/86092/'
+        name: 'HS Riverwood - Sleeping Giant Inn'
+  - name: 'HSRiverwood - Sven and Hilde''s House.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/86094/'
+        name: 'HS Riverwood - Sven and Hilde''s House'
+  - name: 'HSSolitude - Temple of the Divines.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/76056/'
+        name: 'HS Solitude - Temple of the Divines'
+  - name: 'HSSolitude - The Winking Skeever.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/82417/'
+        name: 'HS Solitude - The Winking Skeever'
+  - name: 'HSWhiterun - Arcadia''s Cauldron.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/75340/'
+        name: 'HS Whiterun - Arcadia''s Cauldron'
+  - name: 'HSWhiterun - Jorrvaskr.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/79389/'
+        name: 'HS Whiterun - Jorrvaskr'
+  - name: 'HSWhiterun - The Drunken Huntsman.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/74909/'
+        name: 'HS Whiterun - The Drunken Huntsman'
+  - name: 'HSWilderness - Anise''s Cabin.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/86865/'
+        name: 'HS Wilderness - Anise''s Cabin'
+  - name: 'HSWhiterun - Belethor''s General Goods.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/87317'
+        name: 'HS Whiterun - Belethor''s General Goods'
+  - name: 'HSWhiterun - Dragonsreach.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/88753'
+        name: 'HS Whiterun - Dragonsreach'
+  - name: 'HSWhiterun - Ysolda''s House.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/89227'
+        name: 'HS Whiterun - Ysolda''s House'
+  - name: 'HS(DragonBridge|Markarth|PlayerHomes|Riverwood|Solitude|Whiterun|Wilderness|Winterhold) - (Four Shields Tavern|Silver-Blood Inn|Alvor and Sigrid''s House|Faendal''s House|Hod and Gerdur''s House|Riverwood Trader|Sleeping Giant Inn|Sven and Hilde''s House|Temple of the Divines|The Winking Skeever|Arcadia''s Cauldron|Belethor''s General Goods|Dragonsreach|Jorrvaskr|The Drunken Huntsman|Ysolda''s House|Anise''s Cabin|Breezehome|Honeyside|Proudspire Manor|SeverinManor|Vlindrel Hall)\.esp'
```

Comment:
This is all you're actually adding to the masterlist since we already have everything else, right? You'll need to rebase your PR on the upstream `v0.18` branch.